src/images.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import { sendRequest } from './util';
+
+export default function images(): Object {
+  const baseUrl = 'images-api.nasa.gov';
+  const endpoints = {
+    search: '/search',
+    asset: '/asset',
+    metadata: '/metadata',
+    captions: '/captions',
+  };
+
+  function validateMediaType(mediaType: string): string {
+    const validMediaTypes = ['image', 'audio'];
+    const i = validMediaTypes.indexOf(mediaType.toLowerCase());
+    return i === -1 ? '' : validMediaTypes[i];
+  }
+
+  function validateYear(year: string): boolean {
+    if (!year || typeof year !== 'string') return false;
```

Comment:
we should allow them to pass in a number also, since it will be concatenated into the URL without any issues